src/game.cpp

```diff
@@ -518,7 +518,7 @@ void game::init_ui( const bool resized )
     /**
      * Doing the same thing as above for the overmap
      */
-    static const int OVERMAP_LEGEND_WIDTH = 28;
+    OVERMAP_LEGEND_WIDTH = clamp( TERMX / 5, 28, 55 );
     OVERMAP_WINDOW_HEIGHT = TERMY;
     OVERMAP_WINDOW_WIDTH = TERMX - OVERMAP_LEGEND_WIDTH;
     to_overmap_font_dimension( OVERMAP_WINDOW_WIDTH, OVERMAP_WINDOW_HEIGHT );
```

Comment:
The overmap window can have a different w´font than the rest of the game windows. Have you checked what happens when the overmap font is much larger and what if it the font is much smaller? I assume it's possible the overmap window itself overlays the legend window. Or the legend window is just too much to the left, so it's under the overmap window.

Reply:
> Have you checked what happens when the overmap font is much larger and what if it the font is much smaller?

No, I tested it only on default font dimensions.